From 362c9bee2438ea5f688c7cb0fac2f3f1c884ec0b Mon Sep 17 00:00:00 2001
From: Herbert Xu <herbert@gondor.apana.org.au>
Date: Mon, 1 Sep 2014 13:10:39 -0400
Subject: [crypto] drbg: drbg_exit() can be static

Message-id: <E1XORNX-00089F-Bb@gondolin.me.apana.org.au>
Patchwork-id: 90074
O-Subject: [PATCH 17/19] crypto: drbg - drbg_exit() can be static
Bugzilla: 1011562
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Rafael Aquini <aquini@redhat.com>

RHEL7 Bugzilla #1011562

Upstream commit: 96956aef2ff5521af44d6cb896c967d025360ead

CC: Stephan Mueller <smueller@chronox.de>
Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/crypto/drbg.c b/crypto/drbg.c
index 1fcf322..8527eec 100644
--- a/crypto/drbg.c
+++ b/crypto/drbg.c
@@ -2019,7 +2019,7 @@ static int __init drbg_init(void)
  return crypto_register_algs(drbg_algs, (ARRAY_SIZE(drbg_cores) * 2));
 }
 
-void __exit drbg_exit(void)
+static void __exit drbg_exit(void)
 {
  crypto_unregister_algs(drbg_algs, (ARRAY_SIZE(drbg_cores) * 2));
 }
-- 
1.7.1